plugins/woocommerce/client/legacy/css/auth.scss

```diff
@@ -16,6 +16,7 @@ body {
 
 	img {
 		max-width: 50%;
+		min-height: 50px;
```

Comment:
For some reason this change wasn't included in my build, causing the logo to appear too small to me. With this style it looks fine, though, and the code is good.